borgcron.sh

```diff
@@ -173,7 +182,7 @@ for i in $( seq "$REPEAT_NUM" ); do
 
 	# add local lock
 	do_lock
-
+	
```

Comment:
Huh? 😯 (But it is okay, my IDE would delete this anyway when editing it the next time… :laughing:)

Reply:
👍